### DISCUSSION TITLE
<広報>公式SNSアカウントを作る？

### DISCUSSION THREAD

**Discussion by umepon0626** [tech-creative-club/disaster-posts#29] [General]
ある程度方向性が固まった段階でdisaster-postsの公式SNSアカウントを作って広報するなどして注目を集めていけたら良いのではと感じたんですがどうでしょうか？
災害に関する関心がここ十数年で最も高まっている今，なるべく早いうちから周知を集め
- 資金的インフラ的スポンサー
- 開発を手伝ってくれるエンジニア/デザイナー
- 使ってくれるユーザー
にリーチしていければ，このプロダクトが実現される可能性が高まる気がしました．
Just Ideaなので具体的な進め方等はまだ考えられていないのですが，Postさせていただきます．

**Comment by Ryokugyoku** [comment]
広報活動は、必要そうですね。主要SNSだとFacebookとTwitterとかでしょうか？
ただ、急激に人が増えた形になっていると思うので、意思決定のフローがいまいち見えていない段階で広報活動をするのはちょっと危うい気がするので、一旦組織力の強化が直近の課題な気がします

**Comment by tecsoc** [comment]
> 広報活動は、必要そうですね。主要SNSだとFacebookとTwitterとかでしょうか？
ただ、急激に人が増えた形になっていると思うので、意思決定のフローがいまいち見えていない段階で広報活動をするのはちょっと危うい気がするので、一旦組織力の強化が直近の課題な気がします

おっしゃる通りですね...
